index.html

```diff
@@ -98,6 +98,16 @@
 
   cd "${tmpdir}"
 
+  # Retry downloading several times, as the underlying $get command might not support the retries we need.
+  for i in {1..5}; do
+    echo "Download attempt #${i}..."
+    download_and_install && return 0
+    [ "$i" != 5 ] && sleep "${backoff}"
+    backoff=$(bc <<< $backoff*1.5)
```

Comment:
> backoff=$(bc <<< $backoff*1.5)

Learned something from this, nifty.

Reply:
:smile: 

Unfortunately bash arithmetic only works on integers, so `bc` has to be used.
The "here string" is indeed cool, I also didn't know about it until I found it on stackoverflow :wink: 

There's even a good explanation what its good for https://unix.stackexchange.com/a/80372